doc/tutorials/menu/menu_01.py

```diff
@@ -0,0 +1,38 @@
+"""
+Menu.
+"""
```

Comment:
Since this is included in text, I'd drop the header lines. Or if you want to keep them, expand this to be at least a little more descriptive.

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1608/commits/438501342233c8ff34f90907e395e29fc90d2d9e